src/gl/vbo_mesh.js

```diff
@@ -12,6 +12,7 @@ export default class VBOMesh  {
         this.gl = gl;
         this.vertex_data = vertex_data; // typed array
         this.vertex_layout = vertex_layout;
+        this.vertex_elements = options.vertex_elements;
         this.buffer = this.gl.createBuffer();
```

Comment:
Yeah I think consistency is best. My unconscious habit is to camelCase, so I need to break that. I also default to `var` for the same reason, and we should be consistent with that too.